DataFormats/HcalDetId/src/classes_def.xml

```diff
@@ -5,9 +5,15 @@
   <class name="HcalFrontEndId" ClassVersion="10">
    <version ClassVersion="10" checksum="1241522245"/>
   </class>
-  <class name="HcalDetId" ClassVersion="11">
+  <class name="HcalDetId" ClassVersion="12">
+   <version ClassVersion="12" checksum="1604420376"/>
    <version ClassVersion="11" checksum="1604420376"/>
    <version ClassVersion="10" checksum="193306378"/>
+   <ioread sourceClass = "HcalDetId" version="[-11]" targetClass="HcalDetId" source="uint32_t DetId::id_;" target="DetId::id_">
```

Comment:
when compiling:
```
 WARNING: IO rule for class HcalDetId data member: DetId::id_ was specified as a target in the rule but doesn't seem to appear in target class
```
I suppose, the ioread doesn't work because of that.